frontend/src/metabase/actions/components/ActionViz/Action.unit.spec.tsx

```diff
@@ -248,7 +244,12 @@ describe("Actions > ActionViz > ActionComponent", () => {
 
       userEvent.click(screen.getByRole("button", { name: "Run" }));
 
-      await waitFor(() => expect(scope.isDone()).toBe(true));
+      await waitFor(async () => {
+        const call = fetchMock.lastCall(
```

Comment:
We might find a better pattern for this in the future but it seems that `lastCall` is the only function that gives access to fully constructed `Request` object where it's possible to call `json()`. Other options require manual parsing etc